pkgs/tools/misc/coreutils/default.nix

```diff
@@ -10,6 +10,7 @@
 
 assert aclSupport -> acl != null;
 assert selinuxSupport -> libselinux != null && libsepol != null;
+assert hostPlatform.libc != "msvcrt";
```

Comment:
I think it would make more sense to avoid the assertion and just do ```platforms = platforms.unix``` below. Assertions are annoying because there is no way to override them with something like allowUnsupportedPlatform.

Reply:
fair point.